includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -101,13 +116,294 @@ public function admin_menu_structure() {
 		array_unshift( $submenu['woocommerce'], $menu );
 	}
 
+
 	/**
-	 * Render the checklist
+	 * Returns an array of relevent setup tasks and meta information (completed tasks, uncompleted, etc).
+	 *
+	 * New tasks can be added to the array below. They use the following format:
+	 * title - Task title (e.g. 'Add a product')
+	 * completed_title - Used for the action link once a task is completed (e.g. 'View settings')
+	 * description - A description of the task
+	 * estimate - An estimate in minutes how long the task will take
+	 * link - Destination for the action button
+	 * learn_more - If present, a learn more link will be appended to the description.
+	 * extension - Plugin slug (can be taken from  wp-admin's plugin page). If present, this task will only show when the extension is active.
+	 * condition - A conditional to determine if the task is completed or not.
 	 */
-	public function checklist() {
+	private function get_task_data() {
+		// TODO Double check against setup tasks in the spreadsheet. Needs Canada Post here.
+		// Variables used in the conditions below.
+		$count_posts    = wp_count_posts( 'product' );
+		$total_products = $count_posts->publish;
+		$ups_settings   = get_option( 'woocommerce_ups_settings' );
+		$square_merchant_access_token = get_option( 'woocommerce_square_merchant_access_token' );
+
+		$all_tasks = array(
+			array(
+				'title' => __( 'Add a product', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'Add another product', 'wc-calypso-bridge' ),
+				'description' => __( 'Start by adding your first product to your store.', 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'post-new.php?post_type=product',
+				'condition' => $total_products > 0,
+			),
+
+			array(
+				'title' => __( 'View and customize', 'wc-calypso-bridge' ),
```

Comment:
@justinshreve for the sake of simplificy let's keep the actions as "Do it". 
This is what's being done in the same list for regular sites.